k8s.gcr.io/images/k8s-staging-dns/images.yaml

```diff
@@ -376,3 +388,6 @@
     "sha256:ddb7ee7b6ded62ccfd9cbeaafb87c9eb8b440a0514d0545709c5dd1b5bffbd4f": ["1.22.0"]
     "sha256:5bebd19c21c82a9f19780321f9e6f2b6029299e0f656dc64f5c918d960edca7c": ["1.22.1"]
     "sha256:7b35fdaa9fac8e7e638f6040cd0d49ffe9804dd3f12a0bf124664f1df18e95cf": ["1.22.2"]
+    "sha256:8a2ff9817467f9f2d4559fe5f28664264cf30d8ce71b0da0886e81774c41dd96": ["1.22.3"]
```

Comment:
Humm, this and the following hashes look the same as the ones from `k8s-dns-node-cache` - could you double check?